modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -3393,11 +3726,16 @@ parseGenericData <- function(request) {
   # Create the HTML to display
   htmlSummary <- createHtmlSummary(hasError, allTextErrors, hasWarning, warningList, 
                                    summaryInfo=loadResult$value, dryRun)
-  
+
   if(!dryRun) {
     htmlSummary <- saveAnalysisResults(experiment=experiment, hasError, htmlSummary, loadResult$value$lsTransactionId)
   }
-  
+  # # We don't want to save the dose response html as it can be pretty large so we append it to the html summary
+  # # after save
+  if(!is.null(loadResult$value) && !is.null(loadResult$value$doseResponseHtmlSummary)) {
+    htmlSummary<- paste0(htmlSummary, loadResult$value$doseResponseHtmlSummary)
+  }
+
```

Comment:
Only attach the dose response validation html node when returning as we don't want it saved to the db (too big).